src/renderers/webgl/WebGLTextures.js

```diff
@@ -97,6 +97,17 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 
 	}
 
+	function getInternalFormat( glFormat, glType ) {
+
+		if ( ! _isWebGL2 ) return glFormat;
+
+		if ( glFormat === _gl.RGBA && glType === _gl.FLOAT ) return _gl.RGBA32F;
+		if ( glFormat === _gl.RGBA && glType === _gl.HALF_FLOAT ) return _gl.RGBA16F;
+
+		return glFormat;
```

Comment:
Probably we still need some more work here

https://www.khronos.org/registry/webgl/specs/latest/2.0/#TEXTURE_TYPES_FORMATS_FROM_DOM_ELEMENTS_TABLE